plugins/restapi/v2/plugin.go

```diff
@@ -365,6 +371,15 @@ func configure() {
 		}
 		return restapipkg.JSONResponse(c, http.StatusOK, resp)
 	})
+
+	routeGroup.POST(RouteComputeWhiteFlag, func(c echo.Context) error {
```

Comment:
Can you move the function call between `RouteReceiptsMigratedAtIndex` and `RoutePeer`?
So we have the same order as in iota.go